lib/cyclonedx/report_go_dep_cyclonedx.rb

```diff
@@ -0,0 +1,7 @@
+module Cyclonedx
+  class ReportGoDepCyclonedx < BaseCyclonedx
```

Comment:
Perhaps it's better to rename the class to `ReportGoDep`?  
Otherwise, the class name is really `Cyclonedx:: ReportGoDepCyclonedx`, and the `Cyclonedx` at the end seems redudant.

Reply:
Yep I had it like so for consistency with Sarif `Sarif::BanditSarif` but agree that it's redundant. Happy to rename.